deployments/fargate/README.md

```diff
@@ -5,65 +5,141 @@ for further reading.
 
 Unless stated otherwise, the
 [Splunk OpenTelemetry Connector](https://github.com/signalfx/splunk-otel-collector)
-(Collector) is deployed to an ECS task in a **sidecar** container. We are using the term
-"sidecar" loosely here to mean having the Collector container as a sidecar to multiple or
-all monitored application containers in the task. We do not mean a Collector sidecar container
-per monitored application container as in the strict sense.
+(Collector) is deployed as a **sidecar** (additional container) to ECS tasks.
 
-Requires Connector release v0.30.0 or newer which corresponds to image tag 0.30.0 and newer.
+Requires Connector release v0.31.0 or newer which corresponds to image tag 0.31.0 and newer.
 See image repository [here](https://quay.io/repository/signalfx/splunk-otel-collector?tab=tags).
 
 ## Getting Started
-The default configuration file is located at `/etc/otel/collector/fargate_config.yaml`
-in the Collector image according to the Collector image Dockerfile
-[here](../../cmd/otelcol/Dockerfile). See the contents of the default configuration file
-[here](../../cmd/otelcol/config/collector/fargate_config.yaml). Note that receivers
-`hostmetrics` and `smartagent/ecs-metadata` are specified by default.
+Copy the default Collector container definition JSON below. Replace `MY_SPLUNK_ACCESS_TOKEN`
+and `MY_SPLUNK_REALM` with valid values. Update the image tag to the newest
+version then add the JSON to the `containerDefinitions` section of your task definition
+JSON.
+```json
+{
+  "portMappings": [
+    {
+      "hostPort": 13133,
+      "protocol": "tcp",
+      "containerPort": 13133
+    },
+    {
+      "hostPort": 6060,
+      "protocol": "tcp",
+      "containerPort": 6060
+    },
+    {
+      "hostPort": 14250,
+      "protocol": "tcp",
+      "containerPort": 14250
+    },
+    {
+      "hostPort": 8888,
+      "protocol": "tcp",
+      "containerPort": 8888
+    },
```

Comment:
This doesn't need to be made public, does it?

Reply:
No, it does not.